src/components/Confirm/style.css

```diff
@@ -1,14 +1,41 @@
 @import (less) "../Theme/constants.css";
 
-.title {
-  padding-bottom: 0px;
+.previewsContainer {
+  /*
+  The height of this div needs to be calculated minus the margin top and bottom of its parent
+  This will allow the div to have a consistent height and spacing between its children
+  It will also allow the actions to always sit above the "powered-by-onfido" logo
+   */
+  height: calc(~"100% - 112px");
```

Comment:
is calc necessary, not possible to use margin or padding to achieve the same effect?

Reply:
I was only able to make it work in this way but we can chat about it and find a better solution.

Reply:
just wondering, no worries